.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,52 @@
+name: Bug report
+description: "Create a report to help us improve"
+title: "[BUG]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "### Before you open an issue, please verify if a similar one already exists or has been closed before. More details about the process of contributing can be found in [CONTRIBUTING.md](https://github.com/TheAlgorithms/TypeScript/blob/master/CONTRIBUTING.md)."
```

Comment:
Still not happy with this. I don't want to read this as a part of issues.

Reply:
This is only available for the user _before_ submitting. Once submitted, all Markdown fields are removed.